packages/main/src/TextArea.ts

```diff
@@ -532,6 +532,10 @@ class TextArea extends UI5Element implements IFormElement {
 	}
 
 	_tokenizeText(value: string) {
+		if (!value) {
```

Comment:
It may be better to add that in onBeforeRendering because the `_tokenizeTex`t param type is set to string. This function is called every time on onBeforeRendering anyway.

Reply:
Done.